src/sentry/notifications/platform/slack/provider.py

```diff
@@ -72,8 +75,21 @@ class SlackNotificationProvider(NotificationProvider[SlackRenderable]):
     @classmethod
     def is_available(cls, *, organization: RpcOrganizationSummary | None = None) -> bool:
         # TODO(ecosystem): Check for the integration, maybe a feature as well
+        # I currently view this as akin to a rollout or feature flag for the registry
         return False
 
     @classmethod
     def send(cls, *, target: NotificationTarget, renderable: SlackRenderable) -> None:
-        pass
+        from sentry.integrations.slack.integration import SlackIntegration
+
+        if not isinstance(target, cls.target_class):
+            raise NotificationProviderError(
+                f"Target '{target.__class__.__name__}' is not a valid dataclass for {cls.__name__}"
+            )
+
+        slack_target = PreparedIntegrationNotificationTarget(target=target)
+        installation = SlackIntegration(
+            model=slack_target.integration,
+            organization_id=slack_target.organization_integration.organization_id,
```

Comment:
Is this really all `get_installation` is doing under the hood on the integration class?